scss/utilities/mixins/organisms/_footer.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
```

Comment:
Mixins are intended to be re-used. There seems to be more than necessary here. I think you could probably trim these down to just one or two.